src/BlazorWebFormsComponents.Test/GridView/DataTableSupport.razor

```diff
@@ -3,32 +3,32 @@
 
 <Fixture Test="DataTableAsDataSourceWithAutoGenerateColumns">
 	<ComponentUnderTest>
-		<GridView ItemType="dynamic" DataSource="dataTable" AutogenerateColumns="true">
+		<GridView ItemType="object" DataSource="dataTable" AutogenerateColumns="true">
```

Comment:
I appreciate that this is now `object` ... perhaps for a second pass, after MVP, we could figure out how to default this value to `object` so that it isn't required.

Reply:
Yes that's a good point. I'm not familiar enough with how Blazor handles generic types to figure this out. We'll have to stew on it. For now I see it as an improvement over what was there before (you already had to specify a somewhat arbitrary `ItemType`, now it's just a different type).